modules/src/mock/host.rs

```diff
@@ -60,10 +60,18 @@ impl HostBlock {
     }
 
     pub fn generate_tm_block(chain_id: ChainId, height: u64) -> TmLightBlock {
-        let mut block = TestgenLightBlock::new_default(height).generate().unwrap();
-        block.signed_header.header.chain_id = TMChainId::try_from(chain_id.to_string()).unwrap();
+        // Sleep is required otherwise the generator produces blocks with the
```

Comment:
Thanks for this comment! I don't recall running into problems with the generator being called too fast, but the explanation for the `sleep` here makes sense, and using `new_default_with_time_and_chain_id` instead of `new_default` is definitely an improvement.

That being said, I want to highlight that sleeping 1 second here will slow down tests, so I see some concern here!